engine/schema/src/org/apache/cloudstack/storage/datastore/db/ImageStoreDaoImpl.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.cloudstack.storage.image.db;
```

Comment:
Hi @koushik-das, the reason was to avoid a dependency cycle on Maven poms, I tried explaining it under *Important* section in this PR's description